server/ormconfig.ts

```diff
@@ -35,7 +35,7 @@ function buildConnectionOptions(
     uuidExtension: 'pgcrypto',
     migrationsRun: false,
     logging: data.ORM_LOGGING || false,
-    migrations: ['dist/migrations/**/*{.ts,.js}'],
```

Comment:
It Will only pick up if `dist` is generated which won't be the case when setting up the project